examples/nodejs_etcd_operator/namespace.yaml

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
Do we need to keep the namespace file in the example directory?

Reply:
We are not, the namespace creation/deletion is covered already in the example commons.

Reply:
thanks @pmacik for clarifying